core/config/docs/core.toml

```diff
@@ -103,6 +99,16 @@ SendTimeout = '10s' # Default
 # UseBatchSend toggles sending telemetry to the ingress server using the batch client.
 UseBatchSend = true # Default
 
+[[TelemetryIngress.Endpoints]] # Example
+# Network aka EVM, Solana, Starknet
+Network = 'EVM' # Example
+# ChainID of the network
+ChainID = '111551111' # Example
```

Comment:
just curious, if the network isn't `EVM`, what would the `ChainID` be?

Reply:
For Solana, `mainnet`, `testnet`, `devnet`, or `localnet` - https://github.com/smartcontractkit/chainlink/blob/develop/docs/CONFIG.md#chainid-2